doc/install.rst

```diff
@@ -1,7 +1,10 @@
 Installation
 ------------
+.. warning::
+   Ensure that the :ref:`pre-requisites` from the previous section
+   have been performed, checked and tested before proceeding.
 
-.. note:: The library has been tested against Python 2.7, 3.4, 3.5 and 3.6.
+.. note:: The library has been tested against Python 3.5, 3.6, 3.7 and 3.8.
 
    For **Python3** installation, substitute the following in the
```

Comment:
since python3 is only supported, no need for this section anymore (and rest of doc has to be updated for py3).

Reply:
Removed reference to 2.7 and 3.4.  Added, 3.7 and 3.8.  Removed section that reference the versions of pip, python, etc.